pom.xml

```diff
@@ -7,11 +7,40 @@
     <groupId>groupId</groupId>
     <artifactId>golda-java</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>7</source>
+                    <target>7</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
     <dependencies>
         <dependency>
             <groupId>com.squareup.okhttp3</groupId>
             <artifactId>okhttp</artifactId>
             <version>4.2.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
why do you have few json libs?
each library has its own weight and the project become big no reason.
Also, 2 libs with same name can make strange conflicts - choose the one you are using and remove the other one.

Reply:
removed json.simple (comment it out)